lib/Kconfig.debug

```diff
@@ -2665,19 +2665,19 @@ config LIST_KUNIT_TEST
 	depends on KUNIT
 	default KUNIT_ALL_TESTS
 	help
-	  This builds the linked list KUnit test suite.
-	  It tests that the API and basic functionality of the list_head type
-	  and associated macros.
+    This builds the linked list KUnit test suite.
+    It tests that the API and basic functionality of the list_head type
+    and associated macros.
 
-	  KUnit tests run during boot and output the results to the debug log
-	  in TAP format (https://testanything.org/). Only useful for kernel devs
-	  running the KUnit test harness, and not intended for inclusion into a
-	  production build.
+    KUnit tests run during boot and output the results to the debug log
+    in TAP format (https://testanything.org/). Only useful for kernel devs
+    running the KUnit test harness, and not intended for inclusion into a
+    production build.
 
-	  For more information on KUnit and unit tests in general please refer
-	  to the KUnit documentation in Documentation/dev-tools/kunit/.
+    For more information on KUnit and unit tests in general please refer
+    to the KUnit documentation in Documentation/dev-tools/kunit/.
 
-	  If unsure, say N.
+    If unsure, say N.
```

Comment:
these are supposed to be tabs, not whitespaces (like the other entries in the Kconfig file)